core/domain/fs_domain.py

```diff
@@ -296,12 +296,17 @@ def isfile(self, filepath):
         metadata = self._get_file_metadata(filepath, None)
         return bool(metadata)
 
-    def listdir(self, dir_name):
+    def listdir(self, dir_name, delimiter=None):  # pylint: disable=unused-argument
         """Lists all files in a directory.
 
+        If `delimiter` argument is unused. It is included so that this method
+        signature matches that of other file systems.
+
         Args:
             dir_name: str. The directory whose files should be listed. This
                 should not start with '/' or end with '/'.
+            delimiter: str. It is used to restrict the results to only the
```

Comment:
Don't understand this docstring. How is it used to do so?

Reply:
No it is not used, I figured out. It is not required. Removed.

Reply:
No it isn't used here in the ExplorationFileSystem.
Replaced the description with `Unused`

Reply:
Delimeter removed entirely